watertap/examples/flowsheets/case_studies/electroNP/electroNP_flowsheet.py

```diff
@@ -177,125 +182,95 @@ def build_flowsheet():
-        if "conc_mass_comp[X_ac]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
-        if "conc_mass_comp[X_h2]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
 
     iscale.calculate_scaling_factors(m)
 
     iscale.set_scaling_factor(m.fs.electroNP.properties_byproduct[0.0].flow_vol, 1e7)
     iscale.set_scaling_factor(m.fs.AD.vapor_phase[0].pressure_sat, 1e-3)
 
-    skip_constraint = [
-        "S_O2",
-        "S_N2",
-        "X_AUT",
-        "X_PHA",
-        "S_I",
-        "S_NO3",
-        "S_Mg",
-        "X_MeP",
-        "X_MeOH",
-        "X_PAO",
-        "X_TSS",
-        "S_F",
-        "S_K",
-        "S_A",
-        "X_I",
-        "X_H",
-        "X_PP",
-        "X_S",
-    ]
-
-    for i in skip_constraint:
-        m.fs.electroNP.solute_removal_equation[0.0, "Liq", i].deactivate()
-        m.fs.electroNP.solute_treated_equation[0.0, "Liq", i].deactivate()
-
     m.fs.AD.initialize(outlvl=idaeslog.INFO_HIGH)
     propagate_state(m.fs.stream_adm1_translator)
     m.fs.translator_adm1_asm2d.initialize(outlvl=idaeslog.INFO_HIGH)
     propagate_state(m.fs.stream_translator_electroNP)
     m.fs.electroNP.initialize(outlvl=idaeslog.INFO_HIGH)
     m.fs.costing.initialize()
 
-    solver = get_solver()
-
-    results = solver.solve(m, tee=True)
+    results = solve(m, tee=True)
     return m, results
 
 
+def solve(blk, solver=None, checkpoint=None, tee=False, fail_flag=True):
+    if solver is None:
+        solver = get_solver()
+    results = solver.solve(blk, tee=tee)
+    check_solve(results, checkpoint=checkpoint, logger=_log, fail_flag=fail_flag)
+    return results
+
+
 def display_costing(m):
-    print("\nUnit Capital Costs\n")
+    print("\n----------Capital Cost----------")
+    total_capital_cost = value(
+        pyunits.convert(m.fs.costing.total_capital_cost, to_units=pyunits.USD_2018)
+    )
+    normalized_capex = total_capital_cost / value(
+        pyunits.convert(m.fs.AD.inlet.flow_vol[0], to_units=pyunits.m**3 / pyunits.hr)
+    )
+    print(f"Total Capital Costs: {total_capital_cost:.3f} $")
+    print(f"Normalized Capital Costs: {normalized_capex:.3f} $/m3/hr")
+    print("Capital Cost Breakdown")
     for u in m.fs.costing._registered_unit_costing:
         print(
             u.name,
-            " :   ",
-            value(units.convert(u.capital_cost, to_units=units.USD_2018)),
+            " : {price:0.3f} $".format(
+                price=value(pyunits.convert(u.capital_cost, to_units=pyunits.USD_2018))
+            ),
         )
+    print("\n----------Operation Cost----------")
+    total_operating_cost = value(
+        pyunits.convert(
+            m.fs.costing.total_operating_cost, to_units=pyunits.USD_2018 / pyunits.year
+        )
+    )
+    print(f"Total Operating Cost: {total_operating_cost:.3f} $/year")
+
+    opex_fraction = total_operating_cost / (
+        total_operating_cost + value(m.fs.costing.total_capital_cost)
+    )
```

Comment:
This doesn't make sense to include to me, and I actually removed it from the report.  Also, should the units not work out? That is, $/yr + $ ?

Reply:
This is from our previous 0D flowsheet, we try to report it as an annually Opex proportion

Reply:
I updated the Opex fraction as opex/LCOW